packages/design-system/src/Button/Button.types.ts

```diff
@@ -0,0 +1,38 @@
+import { AriaButtonProps } from "@react-types/button";
+import React from "react";
+
+// Button types
+export type Kind = "primary" | "secondary" | "tertiary" | "error";
+
+// Button sizes
+export type Size = "sm" | "md" | "lg";
```

Comment:
bro what's the use of defining size separately? just define it with buttonprops only and if other components need the same type, you can reference it with `ButtonProps["Size"]`

Reply:
I will keep this size in somewhere common. All our components with size variation should use this.